atomicapp/utils.py

```diff
@@ -352,6 +354,31 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def getLockFolder():
+        if os.access(constants.LOCK_FOLDER, os.W_OK):
```

Comment:
If the current user has write access to `constants.LOCK_FOLDER`, then that one is used, otherwise `constants.CURRENT_TEMP_FOLDER` is used. 

If we do `make test` ( assuming non-root user ), then current user will not have access to `/run/lock/` folder. In this situation lock file is created in `.tmp/...` folder instead of `/run/lock/...`